### DISCUSSION TITLE
installation_config.yml Changes

### DISCUSSION THREAD

**Discussion by uhlhosting** [chatwoot/chatwoot#8478] [Self hosted] [answered]
Hi,

How can we make changes to the values from installation_config.yml, in a caprover deployment, I see some of the values there are showing up in Super Admin console, yet the majority are still missing, privacy links, and such.

How can we make those changes permanent on a docker/caprover deployment?

**Comment by sojan-official** [comment, marked as answer]
We don't recommend editing the values apart from the ones that are exposed in super admin pannel as they are part of the Chatwoot billing, and we don't guarantee persistence. so discretion advised when dealing with these values.
